src/Response/Body.php

```diff
@@ -4,38 +4,29 @@
 
 use Psr\Http\Message\StreamInterface;
 use Jasny\HttpMessage\Stream;
+use Jasny\HttpMessage\Message;
 
 /**
  * ServerRequest body methods
  */
 trait Body
 {
-    /**
-     * 
-     * @var string
-     */
-    protected $defaultStream = 'php://temp';
-    /**
-     * 
-     * @var string
-     */
-    protected $defaultMode = 'w+';
+    use Message\Body;
```

Comment:
To extend a trait, you use the `use` statement. See http://php.net/manual/en/language.oop5.traits.php#language.oop5.traits.composition